tests/TestFramework/PhpUnit/Adapter/PhpUnitAdapterTest.php

```diff
@@ -46,4 +47,21 @@ public function passProvider()
             ['ERRORS!', false],
         ];
     }
+
+    private function getAdapter(): PhpUnitAdapter
```

Comment:
Extracted this method here, similar to [`PhpSpecAdapterTest::getAdapter()`](https://github.com/infection/infection/blob/0.7.0/tests/TestFramework/PhpSpec/Adapter/PhpSpecAdapterTest.php#L93-L102).